README.md

```diff
@@ -161,11 +161,28 @@ nile setup <private_key_alias>
 
 A few things to note here:
 
+- This is a counterfactual deployment, meaning the deployed account pays for its own deployment. You can use [`nile counterfactual-address`](#counterfactual-address) to predict the account address and send the necessary funds beforehand.
 - `nile setup <private_key_alias>` looks for an environment variable with the name of the private key alias.
-- This creates or updates `localhost.accounts.json` file storing all data related to accounts management.
+- This creates or updates `localhost.accounts.json` file storing all data related to account management.
 - The creates or updates `localhost.deployments.txt` file storing all data related to deployments.
 - `--track` and `--debug` flags can be used to watch the status of the account deployment transaction. See `status` below for a complete description.
 
+### `counterfactual-address`
```

Comment:
Is this specifically for UDC? If that's the case, I think we should mention it, and if not, how can you predict the address if the deployer can define the generation mechanism itself? (by changing the salt like UDC).

Reply:
Also is the "counterfactual" part of the command name coming from usual terminology, wouldn't be easier to understand something like "compute-address" (like in [solidity contracts Create2](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/99589794db43c8b285f5b3464d2e0864caab8199/contracts/utils/Create2.sol#L48))

Reply:
as stated a few lines above, this function is to know the address of a counterfactual deployment in advance, to let the user send the funds. it's a specific case of the more general address computation function (defaults to deployed from 0, picks up the signer public key, etc) so i believe leaving the counterfactual word there helps understand the purpose, instead of making it broader ("compute-address").